portfolio/src/main/webapp/index.html

```diff
@@ -7,19 +7,22 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div class="header">
-        <!-- <a href="#default" class="logo">CompanyLogo</a> -->
-        <div class="header-right">
-            <!-- <a class="active" href="#home">Home</a> -->
-            <a href="#about-container" class="active">About</a>
-            <a href="#food-container">Food</a>
-            <a href="#contact-container">Contact</a>
-        </div>
-    </div>
 
     <div id="content">
+        <div class="header">
+            <div class="header-right">
+                <a href="#about-container" class="active" onClick="setActiveTab(this)">About</a>
+                <a href="#food-container" onClick="setActiveTab(this)">Food</a>
+                <a href="#message-container" onClick="setActiveTab(this)">Message</a>
+                <a href="#contact-container" onClick="setActiveTab(this)">Contact</a>
+            </div>
+        </div>
+        <h1 id="title">My Portfolio</h1>
+        <div id="about-comtainer">
+
       <h1 id="title">My Portfolio</h1>
         <div id="about-comtainer" class="container">
+
```

Comment:
nit: Given that the other containers doesn't have the new line before the first header, maybe delete this line too?